.werft/values.fluent-bit.yaml

```diff
@@ -0,0 +1,20 @@
+config:
+  outputs: |
+    [OUTPUT]
+        Name stackdriver
+        Match *
+
+env:
+  - name: GOOGLE_SERVICE_CREDENTIALS
+    value: /gcp/credentials.json
+
+
+extraVolumes:
+  - name: fluent-bit-external
+    secret:
+      secretName: fluent-bit-external
+      defaultMode: 420
+
+extraVolumeMounts:
+  - name: fluent-bit-external
+    mountPath: /gcp
```

Comment:
Since we have some manifests in typescript for harvester stuff, maybe this manifest could also live in `.werft/vm/manifests.ts`?

I believe it'd be nice if we agree on one single way to handle manifests, but no strong opinions!

@mads-hartmann what do you think?

Reply:
I thought `manifests.ts` is for actual resources. This is only a values file like `values.tracing.yaml` which is also in `.werft`.

Reply:
I think this might have to be a `yaml` file as it is passed to helm using `-f` but I'd very much like for it not to be directly in the `.werft` folder ([see RFC](https://www.notion.so/gitpod/Decomposing-the-build-job-2327d3cd8dfa4520b0ad994ab1b49dfd)).

@wulfthimm How about moving this to somewhere else? Maybe dev/charts/fluentbit or .werft/vm/charts/fluentbit.

Alternatively (and maybe this was what you are suggestion @ArthurSens) we could have it be part of the user data and install it during boot that way (in `.werft/vm/manifests.ts`)

Reply:
I wasn't thinking about boot, but I like that idea even better

Reply:
I do not like the idea to add it to the boot process as the serviceAccount secret should be installed before the installation of `fluent-bit`.